### PR TITLE
0.9.0 changes

### PR DESCRIPTION
**Pull request by mikeodr**

### CODE REVIEW

**Comment by mikeodr** [comment]
@mikepurvis could you take a fast when you have a chance, just for sanity?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,52 @@
...
+Publishing to Launchpad
+-----------------------
+
+Better instructions coming soon, but in short:
+
+ - Create a workspace directory.
+
+ - Set an env var with the version you are releasing:
+    ```
+    VER=4.0.3
```
Comment (mikepurvis) [inline README.md]: Heh, update this.

Comment (mikeodr) [inline README.md]: Done.

**Comment by mikepurvis** [comment]
LGTM.

**Pull request merged**